packages/web/components/transactions/transaction-details/transaction-details-content.tsx

```diff
@@ -67,6 +67,16 @@ export const TransactionDetailsContent = ({
     );
   }, [conversion.numerator, conversion.denominator]);
 
+  // if USDC, toggle conversion - temporary until stablecoin logic is implemented
+  useEffect(() => {
```

Comment:
Couldn't this order be determined in the tRPC procedure, also, the conversion rate could be calculated there as well?